parse_test.go

```diff
@@ -17,17 +17,17 @@ import (
 	"github.com/suyashkumar/dicom"
 )
 
-// TestParse is an end-to-end sanity check over DICOMs in testfiles/. Currently it only checks that no error is returned
+// TestParse is an end-to-end sanity check over DICOMs in testdata/. Currently it only checks that no error is returned
 // when parsing the files.
 func TestParse(t *testing.T) {
-	files, err := ioutil.ReadDir("./testfiles")
+	files, err := ioutil.ReadDir("./testdata")
 	if err != nil {
-		t.Fatalf("unable to read testfiles/: %v", err)
+		t.Fatalf("unable to read test files/: %v", err)
```

Comment:
nit: This should be `"unable to read test files:"` (with no trailing slash), but preferably we just say `testdata/:` (since we haven't tried to read the actual test _files_ yet).
```suggestion
		t.Fatalf("unable to read testdata/: %v", err)
```

Reply:
I changed it to `testdata/` as requested.